tests/e2e/numaflowcontroller.go

```diff
@@ -251,7 +254,12 @@ func updateNumaflowControllerRollout(originalVersion, newVersion, pipelineRollou
 		})
 	}
 
-	verifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
+	VerifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
 	// need to track vertices of pipeline spec
-	verifyPipelineRunning(Namespace, pipelineRolloutName, false)
+	if strings.Contains(pipelineRolloutName, "failed") {
+		VerifyPipelineFailed(Namespace, pipelineRolloutName)
+	} else {
+		VerifyPipelineRunning(Namespace, pipelineRolloutName, false)
+	}
```

Comment:
replace this as well

Reply:
I made the change locally for this and pushed it but it's not showing up on Github.. not seen this before.

Reply:
hmm, weird, okay I guess you can do it on a follow-up PR